From 91248f8583b79b4c663273b3d6612bc16abc3e20 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Fri, 6 Dec 2019 14:16:21 +0100
Subject: [PATCH] rpi: Disable image flasher check

Since there's no need to check for
flasher image, we disable this check
so that the command "usb start" is not
executed without reason when booting
from sd-card, IF "usb" is also present
in resin_uboot_device_types array.

Upstream-status: Inappropriate[configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 include/configs/rpi.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/configs/rpi.h b/include/configs/rpi.h
index 8c14faa8e7f2..3b7cb91a5a6f 100644
--- a/include/configs/rpi.h
+++ b/include/configs/rpi.h
@@ -206,7 +206,8 @@
 	ENV_DEVICE_SETTINGS \
 	ENV_DFU_SETTINGS \
 	ENV_MEM_LAYOUT_SETTINGS \
-	BOOTENV
+	BOOTENV \
+	"resin_flasher_skip=1\0"
 
 
 #endif
